pages/index.md

```diff
@@ -6,38 +6,97 @@ PyCon UK is the annual gathering of the UK Python community and its friends from
 This year, we return to [Cardiff City Hall](http://www.cardiffcityhall.com/),
 from Thursday 26th to Monday 30th October.
 
-* * *
+**_Sold Out_**: we are completely sold out with over 700 people coming. We regret
+that we have no waiting list. If you are a sponsor or a speaker with a query about
+tickets please contact the [organising committee](/contact/)    .
 
-Tickets for the main conference have now sold out, and we're expecting over 700 people at the conference!
-(But there are still a handful of tickets left for the [children's day](https://hq.pyconuk.org/children/orders/new/).)
+We [still have tickets](https://hq.pyconuk.org/children/orders/new/) for the 
+[Children's Day](/education/).
 
 * * *
 
-There will be a heavyweight boxing match in Cardiff over the weekend of the
-conference, and so hotel accommodation in the city has been snapped up much
-more quickly this year than usual.  If you're coming to PyCon UK, *book your
-accommodation now*. Don’t delay any further. If you leave it too late, you may
-find that there is no more affordable accommodation available.  Please see the
-[accommodation page](/accommodation/) for more details.
+## Information for Attendees
 
-* * *
+You can review your talk details and the events and meals you've signed up for at 
```

Comment:
Change "talk details" to "ticket details"?

Reply:
Done